src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -65,18 +72,21 @@ def _parse_data(self, data: PandaPowerData, data_type: str, extra_info: Optional
         Returns:
             Converted power-grid-model data
```

Comment:
right now, it always `return`s `self.pgm_input_data`. I think it should return `self.pgm_update_data` when the `data_type == "update"`, right?